File: C++/119. Pascal's Triangle II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> ans;
        int tmp = 1;
        ans.push_back(1);
        for(int i=1;i<=rowIndex;i++)
        {
            long long tmp = 1;
            for(int j=1, k=rowIndex;j<=i;k--,j++)
            {
                tmp *= k;
                tmp /= j;
            }
            ans.push_back(tmp);
        }
        return ans;
    }
};

File: Python/119. Pascal's Triangle II.py
class Solution:
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        ans = [1]
        for i in range(1, rowIndex + 1):
            k,tmp = rowIndex, 1
            for j in range(1, i+1):
                tmp *= k
                tmp //= j
                k -= 1
            ans.append(tmp)
        return ans
            